examples/react-apollo/cypress/support/e2e.ts

```diff
@@ -22,10 +40,54 @@ Cypress.Commands.add('signUpEmailPassword', (email, password) => {
   cy.contains('Continue with email + password').click()
 })
 
+Cypress.Commands.add('signUpEmailPasswordless', (email) => {
+  cy.visit('/sign-up')
+  cy.contains('Continue with passwordless email').click()
+  cy.findByPlaceholderText('Email Address').type(email)
+  cy.contains('Continue with email').click()
+})
+
+Cypress.Commands.add('signInEmailPassword', (email, password) => {
+  cy.visit('/sign-in')
+  cy.contains('Continue with email + password').click()
+  cy.findByPlaceholderText('Email Address').type(email)
+  cy.findByPlaceholderText('Password').type(password)
+  cy.contains('Sign in').click()
+  cy.saveRefreshToken()
+})
+
+Cypress.Commands.add('signInToken', function (path = '/') {
```

Comment:
`visitPathWithRefreshToken` would be a more descriptive name here, could you also please change the second argument to an arrow function? Arrow functions are used elsewhere, we should be consistent.

Reply:
Changed the name, but did not put an arrow function as it requires `this` to get the access token